lib/gollum/app.rb

```diff
@@ -153,7 +153,7 @@ class App < Sinatra::Base
         forbid('Changing this resource is not allowed.')
       end
 
-      post %r{/(deleteFile|rename|edit|create)/custom\.(js|css)} do
+      post %r{/(delete|rename|edit|create)/custom\.(js|css)} do
```

Comment:
Nice catch, basically I had forgotten about the `delete` route in securing the special files... thank! 👍